src/ripple/app/paths/impl/DirectStep.cpp

```diff
@@ -359,8 +375,7 @@ DirectStepI::fwdImp (
 
     bool srcRedeems;
     IOUAmount maxSrcToDst;
-    std::tie (maxSrcToDst, srcRedeems) =
-        maxFlow (sb, src_, dst_, currency_);
+    std::tie (maxSrcToDst, srcRedeems) = maxFlow (sb, in);
```

Comment:
Same comment here as I made in `revImp`. `maxSrcToDst` is compared with `srcToDst`, not `in`.